todo/src/components/TodoList.js

```diff
@@ -12,7 +35,7 @@ class TodoList extends React.Component {
           {this.props.todos.map((todo, index) => (
             <h4
               className={todo.completed ? "completed" : ""}
-              onClick={e => this.toggleCompleted(e, index)}
+              onClick={e => this.props.toggleCompleted(e, index)}
```

Comment:
You can either drop the props on line 38 or drop the e parameter from from the invocation and then the toggleCompleted function on line 25 won't be necessary.